### ISSUE TITLE
API issue between spotweb (dev 1.4.9) and SABNZBd 3.1.1

### ISSUE THREAD

**Issue by MrHappy** [spotweb/spotweb#642]
When I sent a NZB from Spotweb to SABNZBd it works, but show an error-icon, instead of a checkmark.
Also it shows a popup with
`The page at spotweb.domein.eu says:
{'status': True, 'nzo_ids': ['SABnzbd_nzo_ntv4ln82']}`

It seems to resemble [issue 554](https://github.com/spotweb/spotweb/issues/554)

To reproduce:
1. configure sabnzbd in spotweb
2. click a spot to sent to sabnzbd

I expect to see a checkmark and no popup.

**Desktop (please complete the following information):**
 - OS: Android / Linux
 - Browser Firefox / Brave
 - Spotweb Version 1.4.9
 - Development branch
 -  PHP version 7.4
 -  Device information: spotweb on Ubuntu & sabnzbd on NAS.

**Comment by Sweepr** [comment]
I have tried to reproduce this, but here it seems to be working ok, using the 1.4.9 dev branch and 3.1.1 SabNZBd.

Are you seeing this with all NZB's?

**Comment by MrHappy** [comment]
I've tried several and they all gave a message.
Do you use the nzb-key or the api-key?

**Comment by Sweepr** [comment]
I'm using the API-key, but using the NZB-key should also not give these problems.

**Comment by Sweepr** [comment]
Are you 100% sure you are sending the NZB to 3.1.1 SabNZBd and not an older version?

**Comment by MrHappy** [comment]
According to the information I'm running  3.1.1 [99b5a00]

**Comment by Sweepr** [comment]
Could you compaire the file on your installation with this file:
https://github.com/spotweb/spotweb/blob/1aeca9f595bd410a8ac6db27a04f0844a131a9d7/lib/services/NzbHandler/Services_NzbHandler_Pushsabnzbd.php

**Comment by MrHappy** [comment]
Compared them and they differ...
I don't understand, I did a 'git pull' and it told it was up to date...

**Comment by Sweepr** [comment]
That explains why, maybe have a look for a fix here:

https://stackoverflow.com/questions/25411366/git-repo-says-its-up-to-date-after-pull-but-files-are-not-updated

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by Sweepr** [comment]
That explains why, maybe have a look for a fix here:

https://stackoverflow.com/questions/25411366/git-repo-says-its-up-to-date-after-pull-but-files-are-not-updated

**Comment by Sweepr** [comment]
This is the file which contains the fix for compatibilty sabnzbd panel with sabnzbd 3.0.0 https://github.com/spotweb/spotweb/blob/develop/lib/services/NzbHandler/Services_NzbHandler_Pushsabnzbd.php

**Comment by MrHappy** [comment]
Did that, and it says up to date.
However when I run the following the command I get the following response..
`git describe --tags`
`1.4.8-33-g86b6b849`

**Comment by MrHappy** [comment]
> This is the file which contains the fix for compatibilty sabnzbd panel with sabnzbd 3.0.0 https://github.com/spotweb/spotweb/blob/develop/lib/services/NzbHandler/Services_NzbHandler_Pushsabnzbd.php

After the reset the files are identical, but still shows a popup...

**Comment by Sweepr** [comment]
Could it be you are still using a cached version of the Spotweb?  maybe diasble/clear cache/refresh or use a different browser?

For all i can see is that i cannot reproduce so that means there must be some local issue.

**Comment by MrHappy** [comment]
In my nginx logging I see the following line when I sent a nzb to sabnzbd
`[09/Jan/2021:22:46:39 +0100] "GET /?page=getnzb&action=display&messageid=6k5sW02KsRwR7X5XwErTg@spot.net&apikey=<hierstaatuiteraardeenwerkendeapikey> HTTP/1.1" 200 687 "-" "SABnzbd/3.1.1"`
Also cleared cache and tried a browser I never used before (on spotweb) to check this, same issue..

**Comment by Sweepr** [comment]
@meas57 Is there any thing you can think of that could cause this issue?

I cannot reproduce this on my end.

**Comment by mesa57** [comment]
Maybe an invalid spot. You could try to get the nzb of the spot id mentioned.

**Comment by MrHappy** [comment]
> Maybe an invalid spot. You could try to get the nzb of the spot id mentioned.

@mesa57, I've tried several spots and they all result in a popup and an (initial) error-mark behind the spot. The spot gets downloaded just find and when I refresh the page in spotweb the error-mark is replace with a checkmark.

**Comment by MrHappy** [comment]
Found it... After diving into the source code and looking though the userprefs-file I found out I was missing a couple of options on how to send nzb's to sabnzbd. I had only the option in the interface to do nothing or send through the users browser. I was missing 'send through http'. After changing some permissions the option 'send through http' came available and I was able to send nzb's to sabnzbd without any issues.
Thanks for your support and baring with me ;)
